modules/internal/localeModels/flat/LocaleModel.chpl

```diff
@@ -230,7 +145,7 @@ module LocaleModel {
       return chpl_buildLocaleID(numLocales:chpl_nodeID_t, c_sublocid_none);
     }
     proc chpl_name() return local_name();
-    proc local_name() return "rootLocale";
+    proc local_name() return "rootLocale":string;
```

Comment:
I was just making the two locale models identical.  I can delete the \:string from both.